.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: PropelORM CI
+
+on: [
+    push,
+    pull_request
```

Comment:
do we maybe only need one? pull_request?

Reply:
Updated.